### DISCUSSION TITLE
how to limit the upload file size when using minio

### DISCUSSION THREAD

**Discussion by jiangxiaoqiang** [minio/minio#19458] [Q&A]
I am using minio to store some static image with my system. Now I want to limit the upload file(most of them are png/jpg) less than 200KB. Is it possible to limit the file size using minio? or what is the best practice to limit the file size?

**Comment by klauspost** [comment]
S3 does not allow for that. 

Limit the upload form - or maybe some magic with a reverse proxy if you don't trust the client.
